tests/test_item_search.py

```diff
@@ -377,7 +381,9 @@ def test_intersects_results(self):
 
         # Geo-interface object
         class MockGeoObject:
-            __geo_interface__ = intersects_dict
```

Comment:
I don't think this actually tested what it was intended to test before. Anyway, I fixed it so the typechecking works now anyway